src/table/components/head/HeadCellContent.tsx

```diff
@@ -19,12 +19,12 @@ import {
 const HeadCellContent = ({ children, column, isInteractionEnabled, open, setOpen }: HeadCellContentProps) => {
   const { keyboard, translator, changeSortOrder, interactions, embed, app, model } = useContextSelector(
     TableContext,
-    (value) => value.baseProps
+    (value) => value.baseProps,
```

Comment:
This is the opposite of what @a-m-dev have been doing in his PR:s. I think one of you have the wrong version of prettier or the prettier config package